README.md

```diff
@@ -14,6 +14,33 @@ Takeaway Challenge
 
  ```
 
+This program emulates Takeaway restaurant. Customer is able:
+
+* to see the menu 
+* to add several different dishes to the basket 
+* to print the order 
+* to get the total amount
+
+Covered User stories:
```

Comment:
I like that you included the user stories you covered in the README

Reply:
Maybe it would have been better to clear the rest of the README so that it is clearer?